pixi.toml

```diff
@@ -47,6 +47,9 @@ py-test = { cmd = "python -m pytest -vv rerun_py/tests/unit", depends_on = [
   "py-build",
 ] }
 
+# Needs `--target TARGET`
+build-rerun-cli = { cmd = "cargo build --locked -p rerun-cli --no-default-features --features native_viewer,web_viewer --release" }
```

Comment:
This is a bit of an "abstraction leak", because you have to go look at the task in `pixi.toml` to know that you can add `--target TARGET` to the end...

Reply:
That is, unless we have a convention that every one of these inline scripts which are meant to accept arguments are wrapped in a python script with `argparse` for the inputs or something along those lines. But that seems like a lot of work for just accepting some arguments and spawning a process with them 😅

Reply:
I just realised you can do `pixi run <tool>` instead of creating a task for it to run in the context of pixi. I kind of jumped around in the docs so this wasn't obvious to me at first.